src/sfctl/tests/version_test.py

```diff
@@ -0,0 +1,24 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom service command tests"""
```

Comment:
Please update this :)

Reply:
Would it be called just version command test? Does the custom go?

Reply:
I'm okay with version command tests. Thanks!